dashboard/common/db_util.py

```diff
@@ -75,6 +75,9 @@ def get_last_cron_run():
     return datetime.min
 
 def get_canvas_data_date():
+    if not settings.DATA_WAREHOUSE_IS_UNIZIN:
```

Comment:
I feel like this "last cron run" isn't specific to Unizin and this is the only thing this value is used for. This is if the cron is run at all. If you're loading using a separate process and aren't populating the django_cron_cronjoblog then that's when it probably wouldn't be useful.

If you wanted to hide the date or check if the return value was datetime.min and not display it that seems okay. But I think if it didn't return a date it would mess up the base.html temple where last_updated is used.

Reply:
the `DATA_WAREHOUSE_IS_UNIZIN` is currently only used here to avoid the fetch from the`unizin_metadata` table below (currently this is the only unizin specific data that is used from what I can tell). I was looking for a somewhat quick solution so I fell back to what was previously used before the `unizin_metadata` table call was used (which is the call to `get_last_cron_run`).